src/content/docs/Products/SplashKit/Splashkit Expansion/Documentation/onboarding-guide.mdx

```diff
@@ -0,0 +1,229 @@
+
+### Installing Windows Terminal (optional)
+
+Windows Terminal is an updated Command Prompt with many useful features. It is not mandatory to
+install, however it is recommended due to its ease of use. More information about Windows Terminal
+can be found here: [Windows Terminal](https://learn.microsoft.com/en-us/windows/terminal/).It can be
+installed through the Microsoft Store.
+
+By default, new tabs will open as a Command Prompt, with WSL terminals being accessible with the
+down-arrow. Since WSL will be used so frequently, there is the option to change the default tab to
+WSL. Open the settings by clicking the down arrow and selecting ‘Settings’. In the ‘Startup’ tab,
+‘Default profile’ allows you to change the default tab type to WSL.
+
+## Setting Up splashkit-core
+
+Now that WSL is fully configured, it is time to install the splashkit-core repository.
+
+### Forking splashkit-core Repository
+
+A fork is a copy of a repository which allows for independent development without interfering with
+the primary repository itself. To create a personal fork of splashkit-core, navigate to
+[GitHub splashkit-core](https://github.com/splashkit/splashkit-core) and click ‘Fork’. On the next
+page, keep the default name and click ‘Create fork’.
+
+The following guide for the SplashKit Website has useful information regarding forks and branches
+which can be adjusted for splashkit-core by substitutung repository names:
+[Get Your Environment Set Up](https://thoth-tech.netlify.app/products/splashkit/splashkit-website/onboarding/01-setting-up).
+
+### Cloning splashkit-core Repository
+
+Open a WSL terminal and change directory to your home with:
+
+```sh
+cd
+```
+
+Note that this guide clones the repository to the home directory, but feel free to move its location.
+Now initiate the clone process of your fork with:
+
+```sh
+git clone --recursive -j2 https://github.com/{user name}/splashkit-core.git
+```
+
+splashkit-core contains multiple submodules (separate repositories which splashkit-core depends upon).
+The `--recursive` argument ensures that the submodules are also downloaded when calling clone. Wait
+for the download to complete before continuing to the next step.
+
+## Contributing to splashkit-core
+
+It is now time to start fixing bugs and adding functionality to splashkit-core.
+
+### Creating Branch
+
+When modifying the repository, changes should be logically grouped together onto separate branches.
+To create a branch, open a WSL terminal and navigate to the `splashkit-core` folder with:
+
+```sh
+cd
+cd splashkit-core
+```
+
+Check the current branch with:
+
+```sh
+git branch
+```
+
+Create a new branch using the current branch as a base:
+
+```sh
+git branch {new branch name}
+```
+
+Swap to new branch with:
+
+```sh
+git checkout {new branch name}
+```
+
+Now that a new branch is created and active, development can begin.
+
+### Building the Test Programs
```

Comment:
I think it might also be good here to mention the contributing file, and that it has information for these commands in other operating systems (mac and linux)